mscore/scoreaccessibility.cpp

```diff
@@ -412,7 +412,7 @@ std::pair<int, float> ScoreAccessibility::barbeat(Element *e)
             beat = -1;
             ticks = 0;
             }
-      return pair<int,float>(bar + 1, beat + 1 + ticks / static_cast<float>(ticksB));
+      return std::pair<int,float>(bar + 1, beat + 1 + ticks / static_cast<float>(ticksB));
```

Comment:
I suggest `std::make_pair`

Reply:
That would be a semantic change though, this PR is not doing that at all.

Reply:
I saw that too and was tempted to change it, but ultimately I decided against any semantic changes.

Reply:
And if I were making a change here, it would probably be to modern C++ style:

    return { bar + 1, beat + 1 + ticks / static_cast<float>(ticksB) };